pycyphal/transport/udp/_udp.py

```diff
@@ -384,7 +299,7 @@ def _ensure_not_closed(self) -> None:
             raise pycyphal.transport.ResourceClosedError(f"{self} is closed")
 
     def _get_repr_fields(self) -> typing.Tuple[typing.List[typing.Any], typing.Dict[str, typing.Any]]:
-        return [repr(str(self.local_ip_address))], {
+        return [repr(str(self.local_ip_addr))], {
```

Comment:
What do you do now with all of that extra time freed up by not having to type these three extra letters?